packages/bindgen/spec.yml

```diff
@@ -1036,10 +1036,17 @@ classes:
       userAPIKeyProviderClient:
         - sig: () -> UserAPIKeyProviderClient
           cppName: provider_client<app::App::UserAPIKeyProviderClient>
+      push_notification_client: '(service_name: const std::string&) -> PushClient'
       subscribe: '(observer: (app: IgnoreArgument<const App&>)) -> AppSubscriptionToken'
       unsubscribe: '(token: AppSubscriptionToken)'
       call_function: '(user: const SharedSyncUser&, name: std::string, args: EJsonArray, service_name: util::Optional<std::string>, cb: AsyncCallback<(result: Nullable<const EJson*>, err: util::Optional<AppError>)>)'
 
+  PushClient:
+    cppName: app::PushClient
+    methods:
+      register_device: '(registration_token: const std::string&, sync_user: const std::shared_ptr<SyncUser>&, completion: AsyncCallback<(err: util::Optional<AppError>)>&&)'
+      deregister_device: '(sync_user: const std::shared_ptr<SyncUser>&, completion: AsyncCallback<(err: util::Optional<AppError>)>&&)'
```

Comment:
Mentioned this on Slack, but for visibility here as well - the Push Notifications functionality has been deprecated on the server, so it probably doesn't make a whole lot of sense to support it with the bindgen version of the SDK.

Reply:
We are aiming for v12 (SDK-on-bindgen) being as close to a drop-in replacement for v11 (SDK-in-C++) as possible. If we want to remove this for v12, we should deprecate it in v11 ASAP.

Reply:
I think we should - not sure how many people realize this functionality is deprecated on the server, but at some point next year, if they're using it, it'll stop working.